index.html

```diff
@@ -121,7 +121,7 @@ <h3 class="card-header">Profesional Art Printing Data</h3>
                         <li class="boostrap">boostrap</li>
                         <li class="ruby">ruby</li>
                     </ul>
-                    <button  type="button" class="card-btn" id="card-btn-1">Project</button>
+                    <button  type="button" class="card-btn" id="card-btn-1"> See Project</button>
                 </div>
             </div>
             <div class="grid2">
```

Comment:
- [ ] I think it is one of the main requirements is to create the original cards that you already created in HTML you should  create it by using javascript so kindly remove all the code for all the cards from the html and create it by that would help you learning how to create HTML by js 🔥🚀💯

![image](https://user-images.githubusercontent.com/58553711/188764823-a7ac4e1d-31a3-4a48-bf57-e03997c86971.png)

Reply:
ok, I add it in my popup.js File.